bundles/org.openhab.binding.metofficedatahub/src/main/java/org/openhab/binding/metofficedatahub/internal/MetOfficeDataHubSiteApiConfiguration.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.metofficedatahub.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link MetOfficeDataHubSiteApiConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author David Goodyear - Initial contribution
+ */
+@NonNullByDefault
+public class MetOfficeDataHubSiteApiConfiguration {
+
+    public String location = "";
+
+    public int siteSpecificHourlyForecastPollRate = 1;
```

Comment:
```suggestion
    public int hourlyForecastPollRate = 1;
```
I prefer shorter var names and as it is a property of `MetOfficeDataHubSiteApiConfiguration` it is allready obvious that it is site specific and it is better readable in the readme/xml etc.. (this also has to be adjusted there)

Reply:
I went with this as it mapped originally to the specific sub API for the Met Office bits in use, so if others were used it would scope it better. That being said it's unlikely the other API's would be used for OpenHab. So it makes sense to shorten them - have applied to this and the other one - now I've just seen the next comment :) So that's done as well.